Classes/Core/EidRequestBootstrap.php

```diff
@@ -1,6 +1,19 @@
 <?php
 namespace Helhum\AjaxExample\Core;
 
+/*
+ * This file is part of the TYPO3 CMS project.
```

Comment:
uhm, well, this is not correct.

Previously: Copyright: Helmut Hummel, Licence GPL

Now: Copyright TYPO3 CMS, Licence GPL

This is quite a difference